### DISCUSSION TITLE
HDMI Vid Source 2 SB 500 - Help

### DISCUSSION THREAD

**Discussion by socelillyhammer** [GerryDazoo/Slinger#375] [Q&A] [answered]
@GerryDazoo I need some help please regarding the SB 500 on vid source 2 - HDMI.

The way I always had mine setup was with a hdmi decoder copyright box as a passthrough.

When I connect with VLC the video appears for a second and then i get logged off.  Any ideas?  This is the error message I am getting

![Screenshot 2022-11-29 020035](https://user-images.githubusercontent.com/119097385/204461095-3fbdaedf-a275-48a9-9972-57bec1078103.png)

**Comment by socelillyhammer** [comment, marked as answer]
Solved.  had to play around with the frame rates, etc.  sorry to bother everyone.

**Comment by womeara6** [comment]
hI socelillyhammer.  Do you have the specifics on the changes you made.  I am using vid source 2 and have a similar experience.  My cable box has no connections other than hdmi and I had to purchase a converter to set up the composite cable connections.

Thanks.

**Reply by socelillyhammer** [reply]
Sure.  I do have a problem sometimes with connecting with VLC as I have to hit the play button a number of times before I get video.

1. If you are using a slingbox 500 and using the HDMI - you will need the HDMI spliter box:

https://www.amazon.com/OREI-HDMI-Splitter-Out-Duplicate/dp/B005HXFARS/ref=asc_df_B005HXFARS/?tag=hyprod-20&linkCode=df0&hvadid=167172266618&hvpos=&hvnetw=g&hvrand=12853780882451554339&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9004364&hvtargid=pla-308194298374&psc=1

2. assuming you have the correct setup my config coding is as follows

[SLINGBOX]
sbtype="350/500/M1/M2"
password=[put your password]
[if you are doing a remote connect let me know becuase you will need the IP, PORT, and FINDERID]
Resolution=12
FrameRate=60
VideoBandwidth=2000
VideoSmoothness=63
VideoSource = 2 [here i am using HDMI - vid source 2.  if you are using a different vid source change it 0=Composite  1=Component   2=HDMI]

[SERVER]
port=8080 [for now i am using 8080 until i setup my RPi4 as an internal server]
maxstreams=10
enableremote=yes

[REMOTE]
code=2
include=remote.txt 


3. Then if you have that setup edit the runsling.bat file and make the changes as per the notes and readme files

4. then run the slinger

5. then go to vlc media player and your network url should be http://[your ip]:8080/slingbox

Thats it.

Again its all in the readme and notes so Id read through that slowly and step by step

**Reply by GerryDazoo** [reply]
The next time you press play on VLC and don't get video please check the server output to make sure VLC is actually sending a streaming request to the server.
